apps/web/lib/mutations/bookings/create-booking.ts

```diff
@@ -1,10 +0,0 @@
-import * as fetch from "@lib/core/http/fetch-wrapper";
```

Comment:
It's not deleted by moved to `packages/features/bookings/lib/create-booking.ts`.